### PR TITLE
[BS-207][LB] Endpoint for GET individual's details (minimal population)

### PR DESCRIPTION
**Pull request by udinobi**

### CODE REVIEW

File: it/uk/gov/hmrc/breathingspaceifproxy/connector/IndividualDetailsConnectorISpec.scala [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      val url = urlWithoutQuery(IndividualDetailsConnector.path(nino, IndividualDetailsConnector.minimalPopulation))
+      stubCall(HttpMethod.Get, url, Status.BAD_GATEWAY, errorResponsePayloadFromIF, minimalPopulation)
+
+      val response = await(connector.getMinimalPopulation(nino))
+
+      verifyHeaders(HttpMethod.Get, url, minimalPopulation.head)
+      response.fold(_.head.baseError shouldBe SERVER_ERROR, _ => notAnErrorInstance)
+    }
+  }
+}
```
Comment (glidester) [inline it/uk/gov/hmrc/breathingspaceifproxy/connector/IndividualDetailsConnectorISpec.scala:70]: Add a test to check handling a 422 error perhaps

File: app/uk/gov/hmrc/breathingspaceifproxy/connector/IndividualDetailsConnector.scala [hunk 2]
```diff
@@ -16,44 +16,50 @@
...
 import uk.gov.hmrc.breathingspaceifproxy._
 import uk.gov.hmrc.breathingspaceifproxy.config.AppConfig
 import uk.gov.hmrc.breathingspaceifproxy.metrics.HttpAPIMonitor
 import uk.gov.hmrc.breathingspaceifproxy.model._
 import uk.gov.hmrc.http._
+import uk.gov.hmrc.http.HttpReads.Implicits._
 
 @Singleton
-class DebtorDetailsConnector @Inject()(http: HttpClient, metrics: Metrics)(
+class IndividualDetailsConnector @Inject()(http: HttpClient, metrics: Metrics)(
```
Comment (glidester) [inline app/uk/gov/hmrc/breathingspaceifproxy/connector/IndividualDetailsConnector.scala:34]: Should we not keep to our commonly agreed language on the project. These 'Individuals' are debtors in the breathing space context.

File: it/uk/gov/hmrc/breathingspaceifproxy/connector/IndividualDetailsConnectorISpec.scala (hunk 1 above)
Comment (udinobi) [reply it/uk/gov/hmrc/breathingspaceifproxy/connector/IndividualDetailsConnectorISpec.scala:70]: We do not have any specific case in `connector.ConnectorHelper` for 422. It's just included, like any other 4xx error (excluding 404 & 409 which are instead specific), in the general Upstream4xxResponse case. There's no point in asking to WireMock to send 422 and then test that WireMock has actually sent a 422.

File: app/uk/gov/hmrc/breathingspaceifproxy/connector/IndividualDetailsConnector.scala (hunk 2 above)
Comment (udinobi) [reply app/uk/gov/hmrc/breathingspaceifproxy/connector/IndividualDetailsConnector.scala:34]: Yes but our commonly agreed language at this point is `individual`. We have this everywhere in our documentation, even in the API doc.

**Review by glidester — APPROVED** [review]

**Pull request merged**